src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -97,7 +97,21 @@ public virtual async Task<ExecutionResult> ExecuteAsync(ExecutionOptions options
                     }
                     if (document == null)
                     {
-                        document = _documentBuilder.Build(options.Query);
```

Comment:
I believe we need to revert all of the changes within `DocumentExecuter` (which is only the added `try..catch` block).  There already exists proper and sufficient exception handling within the method.